yarn.lock

```diff
@@ -7069,7 +7069,7 @@ npm-user-validate@~1.0.0:
   resolved "https://registry.yarnpkg.com/npm-user-validate/-/npm-user-validate-1.0.0.tgz#8ceca0f5cea04d4e93519ef72d0557a75122e951"
   integrity sha1-jOyg9c6gTU6TUZ73LQVXp1Ei6VE=
 
-npm@6.14.4, npm@^6.10.3:
+npm@^6.10.3:
```

Comment:
This remaining dependency comes from `semantic-release`, so at least it's still a `devDependency`

Reply:
yes, will not in production modules